ios/sandwich/Mappers.swift

```diff
@@ -792,5 +808,38 @@ extension Dictionary where Key == String, Value == Any  {
 
     return Qonversion.ScreenPresentationConfiguration(presentationStyle: presentationStyle, animated: animated)
   }
+  
+  func toNoCodesPresentationConfig() -> NoCodes.PresentationConfiguration {
```

Comment:
I think, NoCodes is extra here
```suggestion
  func toPresentationConfig() -> NoCodes.PresentationConfiguration {
```